src/main/kotlin/org/rust/lang/core/psi/RsCodeFragment.kt

```diff
@@ -19,16 +19,13 @@ import org.rust.lang.RsFileType
 import org.rust.lang.RsLanguage
 import org.rust.lang.core.parser.RustParserUtil.PathParsingMode
 import org.rust.lang.core.parser.RustParserUtil.PathParsingMode.*
-import org.rust.lang.core.psi.ext.RsElement
-import org.rust.lang.core.psi.ext.RsInferenceContextOwner
-import org.rust.lang.core.psi.ext.RsMod
-import org.rust.lang.core.psi.ext.RsNamedElement
+import org.rust.lang.core.psi.ext.*
 import org.rust.lang.core.resolve.Namespace
 
 abstract class RsCodeFragment(
     fileViewProvider: FileViewProvider,
     contentElementType: IElementType,
-    private val context: RsElement
+    open val context: RsElement
```

Comment:
This open/override isn't needed? (this `context` is available via getter PsiElement.getContext())

Reply:
@vlad20012 It is used to provide a setter (see `org.rust.ide.console.RsConsoleCodeFragmentContext#updateContext`)